packages/js/src/workouts/components/ConfigurationWorkout.js

```diff
@@ -384,24 +387,28 @@ export function ConfigurationWorkout( { toggleStep, toggleWorkout, isStepFinishe
 					subtitle={ __( "Tell Google what kind of site you have and increase the chance it gets features in a Google Knowledge Panel. Select ‘Organization’ if you are working on a site for a business or an organization. Select ‘Person’ if you have, say, a personal blog.", "wordpress-seo" ) }
 					isFinished={ isStepFinished( "configuration", steps.siteRepresentation ) }
 				>
+					{  window.wpseoWorkoutsData.configuration.knowledgeGraphMessage &&  <Alert type="warning">
+						{  window.wpseoWorkoutsData.configuration.knowledgeGraphMessage }
+					</Alert> }
+					{ ! state.shouldForceCompany && ! isStepFinished( "configuration", steps.siteRepresentation ) &&
 					<SingleSelect
 						id="organization-person-select"
 						htmlFor="organization-person-select"
 						name="organization"
 						label={ __( "Does you site represent an Organization or Person?", "wordpress-seo" ) }
 						selected={ state.companyOrPerson }
 						onChange={ onOrganizationOrPersonChange }
-						options={ [ {
-							name: "Organization",
-							value: "company",
-						},
-						{
-							name: "Person",
-							value: "person",
-						} ] }
-						readOnly={ isStepFinished( "configuration", steps.siteRepresentation ) }
-
-					/>
+						options={  window.wpseoWorkoutsData.configuration.companyOrPersonOptions }
+					/> }
+					{ (  window.wpseoWorkoutsData.configuration.shouldForceCompany || isStepFinished( "configuration", steps.siteRepresentation ) ) &&
```

Comment:
Here you use `window.wpseoWorkoutsData.configuration.shouldForceCompany` but on [line 393](https://github.com/Yoast/wordpress-seo/pull/17685/files#diff-a0b73e6acef9117beb173fae22d4468a65658118e2b9e3634e71a8b95e7307e3R393) you use `state.shouldForceCompany`. Is this intentional? If not, I think we should keep it consistent.

Reply:
Yeah, I forgot to update line 393. Even if that value goes in the state, we are not changing it so I'd prefer to use the value from the `window` object everywhere.